### ISSUE TITLE
Default Value for actions

### ISSUE THREAD

**Issue by getlarge** [chriskalmar/shyft#146]
Inside an action, i have this attribute : 

```js
  source: {
          type: DataTypeString,
          description: 'the source of truth',
          defaultValue: () => '42',
 },
```

The problem : i can't access to this default value, either in the resolver and the preProcessor payload argument. 

So my temporary solution is to do this inside the resolver :

`payload.source = payload.source || '42';`

Which seems to be duplicated behaviour if we affect a default value ?

What's your opinion on this ?

**Comment by chriskalmar** [comment]
there was indeed a problem with assigning default values on action inputs.
should be fixed now in b067ee4bf0120ca7f7a0bfa5ba8f97dc6495e923

**Comment by getlarge** [comment]
It now works perfectly.
Thanks for the fast fix!
